src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -462,4 +469,14 @@ $(document).ready(function() {
     bindCheckboxToggle();
     var index = $('#new-instructor-index').val();
     bindChangingRole(index);
+
+    if (typeof moment !== 'undefined') {
+        var $selectElement = $('#' + COURSE_TIME_ZONE);
+        TimeZone.prepareTimeZoneInput($selectElement);
+        TimeZone.updateTimeZone($selectElement, courseTimeZone);
+
+        $('#auto-detect-timezone').on('click', function() {
+            autoDetectTimezone();
```

Comment:
If we are standardize time zone to have a space, this can be renamed as `autoDetectTimeZone();`